### ISSUE TITLE
useless-return after del

### ISSUE THREAD

**Issue by dseomn** [PyCQA/pylint#3402]
### Steps to reproduce
Run pylint on this file:

```python
def f(x):
    del x
    return None
```

### Current behavior

`useless-return` is issued.

### Expected behavior

No `useless-return` errors, for the same reason as in #2300.

### pylint --version output

```
pylint 2.4.4
astroid 2.3.3
Python 3.7.6 (default, Jan 19 2020, 22:34:52) 
[GCC 9.2.1 20200117]
```

**Comment by PCManticore** [comment]
Hi, thanks for opening an issue. That's not quite the same as #2300. For #2300 the issue was that a `return None` was more explicit than having a bare `pass` in the body of the function. In your case, the return can be removed since implicitly Python will return a `None` from the function, which is exactly what the check is suggesting.
